networking/kvm.go

```diff
@@ -120,6 +125,15 @@ func setupMacVTapDevice(podID types.UUID, config MacVTapNetConf, interfaceNumber
 	if err := netlink.LinkAdd(link); err != nil {
 		return nil, errwrap.Wrap(errors.New("cannot create macvtap interface"), err)
 	}
+
+	// TODO: duplicate following lines for ipv6 support, when it will be added in other places
+	ipv4SysctlValueName := fmt.Sprintf(IPv4InterfaceArpProxySysctlTemplate, interfaceName)
+	if _, err := cnisysctl.Sysctl(ipv4SysctlValueName, "1"); err != nil {
```

Comment:
For this moment I do not want other behavior than that in `cni`, so IMO this should be discussed in `cni` issue tracker.